code/src/java/plugin/dicebag/DiceBagPlugin.java

```diff
@@ -141,13 +122,13 @@ public void stop()
     @Override
 	public int getPriority()
 	{
-		return SettingsHandler.getGMGenOption(LOG_NAME + ".LoadOrder", 20);
+		return SettingsHandler.getGMGenOption(DiceBagPlugin.LOG_NAME + ".LoadOrder", 20);
```

Comment:
This is an interesting change.  The constant is in this file - why have the fully qualified name?

We also seem to have a mix of tab and space indenting (Override in spaces, public int in tabs) - what is your formatter using?